doc/api/fs.md

```diff
@@ -4361,10 +4358,6 @@ changes:
     description: Passing an invalid callback to the `callback` argument
                  now throws `ERR_INVALID_ARG_TYPE` instead of
                  `ERR_INVALID_CALLBACK`.
-  - version: v14.12.0
-    pr-url: https://github.com/nodejs/node/pull/34993
-    description: The `buffer` parameter will stringify an object with an
-                 explicit `toString` function.
```

Comment:
Hum it looks to me that this does work though, and is still supported 🤔 

```console
$ node -e 'fs.write(1, { toString() { return "string\n" } }, Function.prototype)'
string
```

Reply:
Arf, I'm still falling for it, I forgot that we document that method twice with different signature, nevermind.